custom_components/ramses_cc/remote.py

```diff
@@ -37,23 +70,30 @@ async def async_setup_platform(
 ) -> None:
     """Create remotes for HVAC."""
 
-    if discovery_info is None:  # or not discovery_info.get("remotes")  # not needed
+    if discovery_info is None:
         return
 
-    broker = hass.data[DOMAIN][BROKER]
+    broker: RamsesBroker = hass.data[DOMAIN][BROKER]
 
-    platform = entity_platform.async_get_current_platform()
+    if not broker._services.get(PLATFORM):
+        broker._services[PLATFORM] = True
 
-    for name, schema in SVCS_REMOTE.items():
-        platform.async_register_entity_service(name, schema, f"svc_{name}")
+        platform = entity_platform.async_get_current_platform()
+
+        platform.async_register_entity_service(
+            SVC_LEARN_COMMAND, SVC_LEARN_COMMAND_SCHEMA, "async_learn_command"
```

Comment:
I like consistency...

This service is called `async_*`, but elsewhere they're called `svc_*`.

Make them all one or the other?

Reply:
These are necessarily `async_` as they are what the core service will call and I removed the redundant wrapper. I did consider renaming the others to match so will go that route.

Reply:
Renamed. It reads better this way for the calls that do not originate from a service call so worth doing.